### PR TITLE
chore(react): use handcrafted svg for the infinite loader

### PR DESCRIPTION
**Pull request by elbywan**
#### Crafted an svg replacement for the `<InfiniteLoader />` instead of using a PNG data source.

https://user-images.githubusercontent.com/3428394/145403154-de1130e4-359d-4f50-9db1-e7432bd993a6.mp4

### CODE REVIEW

File: packages/react/src/components/loaders/InfiniteLoader/InfiniteLoader.stories.tsx [hunk 1]
```diff
@@ -1,16 +1,59 @@
...
+\`\`\`
+
+\`\`\`js
+/*
+  Both color and size are optional.
+  Extra props are passed to the child <svg> tag.
+*/
+<InfiniteLoader size={40} color="warning.c50" />
+\`\`\`
+`;
```
Comment (grsoares21) [inline packages/react/src/components/loaders/InfiniteLoader/InfiniteLoader.stories.tsx:23]: Nice!

**Review by grsoares21 — APPROVED** [review]

File: packages/react/src/components/loaders/InfiniteLoader/InfiniteLoader.stories.tsx [hunk 2]
```diff
@@ -1,16 +1,59 @@
...
+const description = `
+### Spinner Component
+
+A simple spinner component with customizable size and color 🎡.
+
+## Usage
+
+\`\`\`js
+
+import { InfiniteLoader } from "@ledgerhq/native-ui"
```
Comment (elbywan) [inline packages/react/src/components/loaders/InfiniteLoader/InfiniteLoader.stories.tsx:13]: note to myself: change this to react-ui 😬

**Review by ofreyssinet-ledger — APPROVED** [review]
nice!

**Review by LFBarreto — APPROVED** [review]
i'll allow it

**Pull request merged**